ide/editor.lib/arch.xml

```diff
@@ -468,6 +468,12 @@ Description of public packages:
         Document property that determines the number of characters in the longest line
         determined during the document loading from a reader by the editor kit. 
     </api>
+    <br/>
+    <api type="export" name="CARET_MIN_WIDTH" category="devel" group="property">
```

Comment:
Category [devel](http://wiki.netbeans.org/API_Stability#Devel) means: _"Under development is a name for a contract that is expected to become a stable API, but that has not yet been finished. The current state serves as a proof of concept, and others are encourage to try it and comment on a dedicated mailing list. Incompatible changes may be done between releases, but should be rare, not radical and properly announced on the mailing list. "_